languages/wp-plugin-hostgator-pt_BR.po

```diff
@@ -549,460 +552,638 @@ msgid "Why choose HostGator for your WordPress site?"
+#: build/2.12.5/index.js:6
 msgid "Proceed"
 msgstr "Continuar"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Production Site"
 msgstr "Site de produção"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Staging Site"
 msgstr "Site de preparação"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "A staging site is a duplicate of your live site, offering a secure environment to experiment, test updates, and deploy when ready."
 msgstr "Um site de preparação é uma duplicata do seu site ativo, oferecendo um ambiente seguro para experimentar, testar atualizações e implantar quando estiver pronto."
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Switch"
 msgstr "Trocar"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Switching"
 msgstr "Trocar"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Switch to Production"
 msgstr "Trocar para produção"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "This will navigate you to the production environment"
 msgstr "Isso o levará ao ambiente de produção"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Loading the production environment now."
 msgstr "Carregando o ambiente de produção agora."
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Switching to the production environment, this should take about a minute."
 msgstr "Alternando para o ambiente de produção, isso deve levar cerca de um minuto."
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Switch to Staging"
 msgstr "Alternar para o ambiente de preparação"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "This will navigate you to the staging environment"
 msgstr "Isso o levará ao ambiente de preparação"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Loading the staging environment now."
 msgstr "Carregando o ambiente de preparação agora."
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Switching to the staging environment, this should take about a minute."
 msgstr "Alternando para o ambiente de preparação, isso deve levar cerca de um minuto."
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "An unknown error has occurred."
 msgstr "Ocorreu um erro desconhecido."
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "Working…"
 msgstr "Funcionando…"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:1
 msgid "Sales Channel"
 msgstr "Canal de vendas"
 
-#: build/2.12.4/index.js:1
+#: build/2.12.5/index.js:6
 msgid "WonderBlocks provides a library of customizable block patterns and page templates."
 msgstr "O WonderBlocks fornece uma biblioteca de padrões de blocos e modelos de página personalizáveis."
+
+#: build/2.12.5/index.js:1
+msgid "Asks the browser to download and cache links on the page ahead of them being clicked on, so that when they are clicked they load almost instantly. "
```

Comment:
@circlecube I was testing this and noticed that we’ve lost the translations for the Link Prefetch and Jetpack Boost strings from the original PR: https://github.com/newfold-labs/wp-plugin-hostgator/pull/853/files Let me know if you want me/someone from the team to add them back again.

I also created one more bump PR that we’re hoping to include in this release. Hope that’s OK: https://github.com/bluehost/bluehost-wordpress-plugin/pull/1495 Thanks! 😄

Reply:
Thanks for calling this out @arunshenoy99, I'm looking at bringing these strings back into this release now.